eras/alonzo/impl/cddl-files/alonzo.cddl

```diff
@@ -1,106 +1,336 @@
+; 
+auxiliary_data = metadata
+                  / [transaction_metadata : metadata
+                    , auxiliary_scripts : auxiliary_scripts]
+                  / #6.259({? 0 : metadata
+                           , ? 1 : [* native_script]
+                           , ? 2 : [* plutus_script]})
+
+auxiliary_data_hash = $hash32
+
+auxiliary_scripts = [* native_script]
+
+big_int = int / big_uint / big_nint
+
+big_nint = #6.3(bounded_bytes)
+
+big_uint = #6.2(bounded_bytes)
+
+; Valid blocks must also satisfy the following two constraints:
+;   1) the length of transaction_bodies and transaction_witness_sets must be
+;      the same
+;   2) every transaction_index must be strictly smaller than the length of
+;      transaction_bodies
+; NEW: 
+;   invalid_transactions
+; 
+block = [header
+        , transaction_bodies : [* transaction_body]
+        , transaction_witness_sets : [* transaction_witness_set]
+        , auxiliary_data_set : {* transaction_index => auxiliary_data}
+        , invalid_transactions : [* transaction_index]]
+
+bootstrap_witness = [public_key : $vkey
+                    , signature : $signature
+                    , chain_code : bytes .size 32
+                    , attributes : bytes]
+
+bounded_bytes = bytes .size (0 .. 64)
+
+certificate = [stake_registration
+               // stake_deregistration
+               // stake_delegation
+               // pool_registration
+               // pool_retirement
+               // genesis_key_delegation
+               // move_instantaneous_rewards_cert]
+
+coin = uint
+
+; NOTE: NEW
+;   The keys to the cost model map are not present in the serialization.
+;   The values in the serialization are assumed to be ordered
+;   lexicographically by their correpsonding key value.
+;   See Plutus' `ParamName` for parameter ordering
+; 
+cost_model = [166*166 int64]
+
+cost_models = {* language => cost_model}
+
+credential = [0, addr_keyhash // 1, script_hash]
+
+; This too has been introduced in Shelley as a backport from Alonzo.
+; 
+delta_coin = int
+
+dns_name = text .size (0 .. 64)
+
+epoch = uint
+
+ex_unit_prices = [mem_price : positive_interval, step_price : positive_interval]
+
+ex_units = [mem : uint, steps : uint]
+
+genesis_delegate_hash = $hash28
+
+genesis_hash = $hash28
+
+header = [header_body, body_signature : $kes_signature]
+
+;  block_body_size: merkle triple root
+; 
+header_body = [block_number : uint
+              , slot : uint
+              , prev_hash : $hash32 / nil
+              , issuer_vkey : $vkey
+              , vrf_vkey : $vrf_vkey
+              , nonce_vrf : $vrf_cert
+              , leader_vrf : $vrf_cert
+              , block_body_size : uint
+              , block_body_hash : $hash32
+              , operational_cert
+              , protocol_version]
+
+int64 = -9223372036854775808 .. 9223372036854775807
+
+ipv4 = bytes .size 4
+
+ipv6 = bytes .size 16
+
+; NOTE: NEW
+;   This is an enumeration. for now there's only one value
```

Comment:
Missing the Plutus v1 part of the comment which is maybe more useful

Reply:
Nice catch! Thanks! :+1: